Microsoft.Identity.Web/WebApiServiceCollectionExtensions.cs

```diff
@@ -38,20 +38,18 @@ public static class WebApiServiceCollectionExtensions
             this IServiceCollection services,
             IConfiguration configuration,
             X509Certificate2 tokenDecryptionCertificate = null,
-            string configSectionName = "AzureAD",
+            string configSectionName = "AzureAd",
             bool subscribeToJwtBearerMiddlewareDiagnosticsEvents = false)
         {
             services.AddAuthentication(AzureADDefaults.JwtBearerAuthenticationScheme)
                     .AddAzureADBearer(options => configuration.Bind(configSectionName, options));
+            services.Configure<AzureADOptions>(options => configuration.Bind(configSectionName, options));
 
             services.AddHttpContextAccessor();
 
             // Change the authentication configuration to accommodate the Microsoft identity platform endpoint (v2.0).
             services.Configure<JwtBearerOptions>(AzureADDefaults.JwtBearerAuthenticationScheme, options =>
             {
-                // Reinitialize the options as this has changed to JwtBearerOptions to pick configuration values for attributes unique to JwtBearerOptions
-                configuration.Bind(configSectionName, options);
```

Comment:
>configuration [](start = 16, length = 13)

removing this fails to pick JwtBreareerOptions ([https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.builder.jwtbeareroptions?view=aspnetcore-1.1](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.builder.jwtbeareroptions?view=aspnetcore-1.1)) property like JwtBearerOptions.Audience.

Reply:
I tested and confirm that it doesn't pick values that you set on appsettings. 

An easy way to repro this issue is by running the step 4.1 (call own web api), set the the property `"Audience":  "test"` in the `appsettings.json` and set a breakpoint on line 51. You will notice that `options.Audiance` didn't change. 

Adding `configuration.Bind(configSectionName, options);` back fix this problem